sdk/python/tests/test_historical_retrieval.py

```diff
@@ -174,6 +177,10 @@ def get_expected_training_df(
                 f"driver_stats__{k}": driver_record.get(k, None)
                 for k in ("conv_rate", "avg_daily_trips")
             }
+            if full_feature_names
+            else {
+                k: driver_record.get(k, None) for k in ("conv_rate", "avg_daily_trips")
```

Comment:
I think it'll be cleaner if you handle if/else only with k / f"driver_stats__{k}":
```python
formatted_k = f"driver_stats__{k}" if full_feature_names else k
order_record.update(
  {
      formatted_k: driver_record.get(k, None)
      for k in ("conv_rate", "avg_daily_trips")
  }
```